Sources/generate-swift-syntax-builder/gyb_syntax_support/AttributeKinds.py

```diff
@@ -0,0 +1,871 @@
+from .Child import Child
+from .Node import Node  # noqa: I201
+
+class Attribute(object):
+    def __init__(self, name, swift_name=None):
+        self.name = name
+        self.swift_name = swift_name or name
+
+    def macro_value(self):
+        return ""
+
+class TypeAttribute(Attribute):
+    def __init__(self, name):
+        super().__init__(name, name)
+
+    def macro_value(self):
+        return "TYPE_ATTR(" + self.name + ")"
+
+class DeclAttribute(Attribute):
+    def __init__(self, name, klass, *options, code, swift_name=None):
```

Comment:
What do you think about moving `code` and `swift_name` before `options`? That way the variadic options are the last option, which makes it easier to read IMHO.

Reply:
I like having the code last. It helps it stand out, and python requires you spell `code=` after the positional arguments here.

Reply:
I don’t have super strong opinions, so fine with me.

Reply:
What about `swift_name`. I think it makes more sense to move that after `name`.